addons/website/views/snippets/s_popup.xml

```diff
@@ -34,7 +35,7 @@
 
 <template id="s_popup_options" inherit_id="website.snippet_options">
     <xpath expr="." position="inside">
-        <div data-js="SnippetPopup" data-selector=".s_popup" data-target=".modal">
+        <div data-js="SnippetPopup" data-selector=".s_popup:not('#website_cookies_bar')" data-target=".modal">
```

Comment:
Not sure why? The whole point of using s_popup structure for the cookie bar was to have the option working on the cookie bar too :)